### ISSUE TITLE
Crash when ticking "Show fiat balances for addresses" option

### ISSUE THREAD

**Issue by simon-v** [fyookball/electrum#325]
Python traceback in the terminal:
```
Traceback (most recent call last):
  File "{...}/Electron_Cash/gui/qt/main_window.py", line 2714, in on_fiat_address
    self.address_list.refresh_headers()
AttributeError: 'AddressList' object has no attribute 'refresh_headers'
```

**Comment by jamesmorgan** [comment]
This happened to me, V3.0 mac osx

**Comment by TheButterZone** [comment]
Ditto.

**Comment by TheSentry** [comment]
Happened to me, too. Windows 10, v3.0

**Comment by kyuupichan** [comment]
Fixed by b03a80cec75f261003d21862426f9d4ffe17575f
